main/lsp/notifications/sorbet_workspace_edit.cc

```diff
@@ -75,12 +75,7 @@ void SorbetWorkspaceEditTask::run(LSPTypecheckerDelegate &typechecker) {
         Exception::raise("Attempted to run a slow path update on the fast path!");
     }
     const auto newEditCount = updates->editCount - updates->committedEditCount;
-    typechecker.typecheckOnFastPath(move(*updates));
-    if (latencyTimer != nullptr) {
-        ENFORCE(newEditCount == params->diagnosticLatencyTimers.size());
```

Comment:
Can we add the following ENFORCE?

```
ENFORCE(latencyTimer == nullptr || newEditCount == params->diagnosticLatencyTimers.size());
```

This checks in debug builds that we have exactly 1 diagnostic latency timer per edit.